gwt/src/main/resources/org/pentaho/platform/dataaccess/datasource/DatasourceEditor.gwt.xml

```diff
@@ -28,6 +28,8 @@ This module serves as the Debuging environment and default implementation for in
   <inherits name="org.pentaho.ui.database.DatabaseDialog"/>
   <inherits name="org.pentaho.database.Database"/>
   <inherits name="org.pentaho.reporting.libraries.base.Util"/>
+  <inherits name="org.pentaho.mantle.login.MantleLogin"/>
+  <inherits name="org.pentaho.mantle.SchedulingDialogs"/>
```

Comment:
We need to include all of this to be able to use `CsrfUtil`?

Reply:
gwt doesn't compile without it

Reply:
It's strange that you need two includes, no? We depend on two classes, of the same package.

Reply:
I'll double check it

Reply:
@dcleao the need of having this 2 inherits to compile data-access gwt module, is related to the way commons-gwt project was built. Both CSrfUtil and JsrCsrfToken are included on that project.

The lines that are currently needed are:
`<inherits name="org.pentaho.mantle.login.MantleLogin"/>
  <inherits name="org.pentaho.mantle.SchedulingDialogs"/>`

They are needed because they are refereeing both gwt module config paths of commons-gwt module. And those config files are:
- https://github.com/pentaho/pentaho-commons-gwt-modules/blob/master/widgets/src/main/resources/org/pentaho/mantle/SchedulingDialogs.gwt.xml
- https://github.com/pentaho/pentaho-commons-gwt-modules/blob/master/widgets/src/main/resources/org/pentaho/mantle/login/MantleLogin.gwt.xml

This "inherits" also exists for changes that we already performed in other projects during Marketo working:
- https://github.com/pentaho/pentaho-platform/blob/master/user-console/src/main/resources/org/pentaho/mantle/MantleApplication.gwt.xml#L25

To change this, probably we should invest some more time on refactoring commons-gwt widgets modules to expose also CsrfUtil and JsCsrfToken packages along with MantleLogin and SchedulingDialogs.

This is my quick theory after a fast research, this will require further investigation. Could be a TODO work to be added to our list.

Reply:
Ok, thanks. Move on :-)

Reply:
I will send a cleanup PR because it's just needed 1 of the inherits, I just figured that out during the double check :)